src/openms/include/OpenMS/FORMAT/FileTypes.h

```diff
@@ -95,11 +95,28 @@ namespace OpenMS
 
     enum class FileProperties
     {
-      READABLE,
-      WRITEABLE,
-      PROVIDES_MS1,
-      PROVIDES_MS2,
-      SIZE_OF_FILEPROPERTIES
+      READABLE,                         // SOMETHING in OpenMS can read this (it doesn't have to be in FileHandler though)
+      WRITEABLE,                        // SOMETHING in OpenMS can write this (it doesn't have to be in FileHandler though) 
+      PROVIDES_MS1,                     // File CAN contain spectra with no parent
```

Comment:
don't  undestand the comment. what is the parent

Reply:
if PROVIDES_MS1 is set? don't get it

Reply:
For if the files can contain both MS1 and MS2. I included these to handle cases like DTA and MS2, but now having looked at all our supported formats, this looks to only apply to them. I'm always happy to simplify and take these out if you think it's not necessary.